src/development/stack.yml

```diff
@@ -344,6 +350,162 @@ services:
       - postgres_user
     volumes:
       - ../../../reccoom/:/srv/app/
+
+  zammad-elasticsearch:
+    image: bitnami/elasticsearch:${ELASTICSEARCH_VERSION:-8.18.0}
+    restart: always
```

Comment:
`restart` does not exist for `stack` I believe, see `deploy.restart_policy.condition` (https://docs.docker.com/reference/compose-file/deploy/#restart_policy)